docs/DeveloperGuide.md

```diff
@@ -332,33 +333,43 @@ As a result, any updates to `ObservableList<Resident>` would be reflected immedi
 
 ### Show/hide feature for resident fields
 
-**Challenges faced with UI:**
-
-The original UI represented a `Person` field as a nested FXML `Label` within a `PersonCard`, which was used to populate the `ListView` panel. Calling `setVisible` on a `Label` resulted in blank gaps in the panel because the `Label` was ultimately still intact, just *invisible*. Hence, there was a need to find another method to hide and collapse rows/columns.
+**Changes to UI component:**
 
-**Gaps in** `ListView` **panel:**
+To achieve the show/hide feature for resident fields, we first had to rework our UI to use a `TableView`
+and two `ObservableList<String>` objects, instead of a `ListView` and a `PersonCard`. This was because the `setVisible`
+method does not work properly with a `ListView` and a `PersonCard` -- as shown below, hidden fields would be rendered 
+as gaps in the `ListView`, rather than being omitted entirely. 
```

Comment:
I thought we should not mention any past implementations, i.e. `ListView` and `PersonCard`

Reply:
Hi Neale, I've removed this section on the UI implementation, since you have also mentioned it in your PR.